CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [0.63.48](https://github.com/brave/brave-browser/releases/tag/v0.63.48)
+
+ - Added Brave Ads. ([#4209](https://github.com/brave/brave-browser/issues/4209))
+ - Added ability to clear browsing data when exiting Brave. ([#492](https://github.com/brave/brave-browser/issues/492))
+ - Added prompt when websites request Widevine permissions. ([#1940](https://github.com/brave/brave-browser/issues/1940))
+ - Added global navigation bar under brave://settings. ([#956](https://github.com/brave/brave-browser/issues/956))
+ - Added brave://rewards-internals to assist with Rewards support. ([#1174](https://github.com/brave/brave-browser/issues/1174))
+ - Updated buttons throughout Brave to match product branding. ([#3378](https://github.com/brave/brave-browser/issues/3378))
+ - Updated checkboxes throughout Brave to match Brave color palette. ([#3377](https://github.com/brave/brave-browser/issues/3377))
+ - Fixed characters being removed from URL when typing into omnibox. ([#3756](https://github.com/brave/brave-browser/issues/3756))
+ - Fixed intermittent hang on mobile.twitter.com when right clicking the page. ([#2410](https://github.com/brave/brave-browser/issues/2410))
+ - Fixed not being able to auto-contribute even though the required BAT is available. ([#3897](https://github.com/brave/brave-browser/issues/3897))
+ - Fixed auto-contribution table remains corrupted after upgrading Brave in certain cases. ([#3510](https://github.com/brave/brave-browser/issues/3510))
+ - Fixed YouTube publisher information not being displayed under rewards panel in certain cases. ([#3335](https://github.com/brave/brave-browser/issues/3335))
+ - Fixed favicon for verified publishers not being displayed correctly under rewards panel. ([#3783](https://github.com/brave/brave-browser/issues/3783))
+ - Fixed special characters in YouTube channel names not being displayed correctly under rewards panel.([#2028](https://github.com/brave/brave-browser/issues/2028))
+ - Fixed attention percentage not immediately being updated under rewards panel. ([#3497](https://github.com/brave/brave-browser/issues/3497))
+ - Fixed auto-contribution table not being updated under Rewards in certain cases. ([#2911](https://github.com/brave/brave-browser/issues/2911))
+ - Fixed not being able to exclude sites from auto-contribution table under Rewards. ([#2943](https://github.com/brave/brave-browser/issues/2943))
+ - Fixed translations on the rewards panel for various locales. ([#3426](https://github.com/brave/brave-browser/issues/3426))
+ - Fixed restored publishers being removed from auto-contribution table when brave://rewards is opened in multiple windows. ([#3264](https://github.com/brave/brave-browser/issues/3264))
+ - Fixed publishers that are excluded are toggled back as included in certain cases. ([#3551](https://github.com/brave/brave-browser/issues/3551))
```

Comment:
Fixed publishers that are excluded being toggled back to included in certain cases.